packages/mgt-components/src/components/mgt-teams-channel-picker/mgt-teams-channel-picker.graph.ts

```diff
@@ -18,3 +25,43 @@ export async function getAllMyTeams(graph: IGraph): Promise<Team[]> {
   const teams = await graph.api('/me/joinedTeams').select(['displayName', 'id', 'isArchived']).get();
   return teams ? teams.value : null;
 }
+
+/** An object collection of cached photos. */
+type CachePhotos = {
+  [key: string]: CachePhoto;
+};
+
+export async function getTeamsPhotosforPhotoIds(graph: BetaGraph, teamIds: string[]): Promise<CachePhotos> {
+  let cache: CacheStore<CachePhoto>;
+  let photos: CachePhotos = {};
+
+  if (getIsPhotosCacheEnabled()) {
+    cache = CacheService.getCache<CachePhoto>(schemas.photos, schemas.photos.stores.teams);
+    for (const id of teamIds) {
+      try {
+        const photoDetail = await cache.getValue(id);
```

Comment:
@vogtn When adding the caching, I got some issues initializing this new store I added. I had to log out, login and then refresh. Is this expected behavior? I didn't face it when I did the caching for the generic picker.